code/game/turfs/open/floor.dm

```diff
@@ -161,14 +161,19 @@
 	W.update_icon()
 	return W
 
-/turf/open/floor/attackby(obj/item/C, mob/user, params)
-	if(!C || !user)
-		return 1
-	if(..())
-		return 1
-	if(intact && istype(C, /obj/item/stack/tile))
-		try_replace_tile(C, user, params)
-	return 0
+/turf/open/floor/attackby(obj/item/object, mob/user, params)
+	if(!object || !user)
+		return TRUE
+	. = ..()
+	if(.)
+		return TRUE
```

Comment:
```suggestion
		return
```
If this is truthy, let it pass whatever return the parent sent.

Reply:
yes i have no idea why i did not think of that.